reversion/models.py

```diff
@@ -62,6 +62,19 @@ class Revision(models.Model):
         help_text="A text comment on this revision.",
     )
 
+    def __repr__(self):
+        date_created = self.date_created
+        if date_created:
+            iso_date_created = date_created.replace(microsecond=0).isoformat()
+        else:
+            iso_date_created = None
+        return 'Revision(pk={!r}, comment={!r}, date_created={!r}, user={!r})'.format(
+            self.pk,
+            self.comment,
+            iso_date_created,
```

Comment:
I think we can just use the built-in repr of date_created here.

Reply:
It is a bit long though - but has the benefit of being simpler.
Will change it.